src/batou/environment.py

```diff
@@ -247,6 +251,19 @@ def load_environment(self, config):
             sandbox = getattr(batou.vfs, sandbox)(self, config["vfs"])
             self.vfs_sandbox = sandbox
 
+        def parse_addr_option(value):
+            if value == "optional":
+                return value
+            return bool(ast.literal_eval(value))
+
+        for attr in ["require_v4", "require_v6"]:
+            if attr not in environment:
+                continue
+            setattr(self, attr, parse_addr_option(environment[attr]))
+
+        batou.utils.Address.require_v4 = self.require_v4
```

Comment:
That's an "interesting" way to do it. I can live with it but maybe we want to refactor this later?